homeassistant/const.py

```diff
@@ -1349,6 +1349,7 @@ class UnitOfPrecipitationDepth(StrEnum):
 CONCENTRATION_MILLIGRAMS_PER_CUBIC_METER: Final = "mg/m³"
 CONCENTRATION_MICROGRAMS_PER_CUBIC_FOOT: Final = "μg/ft³"
 CONCENTRATION_PARTS_PER_CUBIC_METER: Final = "p/m³"
+CONCENTRATION_GRAMS_PER_LITER: Final = "g/L"
```

Comment:
This adds new global units to Home Assistant. This requires an architectural discussion.

Alternatively, don't add this globally.